landscaper/collector/physical_host_collector.py

```diff
@@ -60,10 +61,26 @@ def init_graph_db(self):
         """
         LOG.info("Adding physical machines to the landscape...")
         now_ts = time.time()
-        for machine in self.conf_mgr.get_machines():
-            self._add_physical_machine(machine, now_ts)
+        machines = self.conf_mgr.get_machines()
+        self._add_physical_machine_threads(machines, now_ts)
         LOG.info("Finished adding physical machines to the landscape.")
 
+    def _add_physical_machine_threads(self, machines, timestamp):
```

Comment:
Just a small comment, _add_physical_machine function will look better without _threads postfix. To keep name consistency will be better to modify existing function and add treading option as a parameter or rename threading function or even create new class.